### PR TITLE
Swipers

### PR DESCRIPTION
**Pull request by CristobalMe**

### CODE REVIEW

File: frontend/src/components/swiper/CoverFlow.jsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      "numReviews": 100,
+      "id": 10
+    }
+  ]
+
+
+
+  return (
+    <div className='w-[30rem] h-[20rem]'>
+      <Swiper
```
Comment (toriqyanM) [inline frontend/src/components/swiper/CoverFlow.jsx:106]: Unfortunately swiper would not be a fair game for complex UI design in this case. The goal for the requirement is for you to demonstrate your ability to achieve complicated UI design with css and html. Usage of this module would likely put you in risk of not showcasing your own skill.

File: frontend/src/components/swiper/CoverFlowCard.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+const CoverFlowCard = ({data}) => {
+    return(
+      <div className="max-w-sm rounded overflow-hidden shadow-lg bg-white h-[20rem] w-[120rem] bg-center bg-cover">
+        <img className="h-[10rem] w-full" src={data.imageUrl} alt="https://www.shutterstock.com/image-vector/default-ui-image-placeholder-wireframes-600nw-1037719192.jpg" />
+        <div className="px-3 py-4 flex flex-col justify-center items-center">
+          <div className="font-bold text-base mb-2 mx-[1.3rem]">{data.name}</div>
+          <p className="text-gray-700 text-sm mx-[1.1rem]">
+            📍{data.location}
+          </p>
```
Comment (toriqyanM) [inline frontend/src/components/swiper/CoverFlowCard.jsx:11]: Is this your code? I would be very careful about directly copy pasting here. If you do, do cite them. And also note that, you will be evaluated on how much code YOU write.

**Review by toriqyanM — CHANGES_REQUESTED** [review]
I have concerns about not enough code by your self to achieve the complex UI requirement.

File: frontend/src/components/swiper/CoverFlowCard.jsx (hunk 2 above)
Comment (CristobalMe) [reply frontend/src/components/swiper/CoverFlowCard.jsx:11]: Yes, this part is mine

**Comment by toriqyanM** [comment]
Abandon this PR if you don't plan to keep working on it.

File: frontend/src/components/swiper/CoverFlow.jsx (hunk 1 above)
Comment (CristobalMe) [reply frontend/src/components/swiper/CoverFlow.jsx:106]: To Do: 
complex UI design

**Pull request closed without merging**